### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vmisai**
[DEMO LINK](https://vmisai.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/index.html
Comment (etojeDenys) [inline src/index.html:16]: remove all comments

File: src/styles/clock.scss
Comment (etojeDenys) [inline src/styles/clock.scss:72]: why do we need to the same keyframes

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/styles/clock.scss
Comment (maxim2310) [inline src/styles/clock.scss:72]: remove this one and use just rotatesecond everywhere

File: src/styles/clock.scss
Comment (maxim2310) [inline src/styles/clock.scss:51]: and you need to play animation unstoppable

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

**Review by maxim2310 — APPROVED** [review]
Good work!!!

**Pull request closed without merging**
